server/api/index.js

```diff
@@ -15,24 +15,47 @@ class Entity {
     }
   }
 
-  async getById() {}
+  async getById(req, res, entityName) {
+    try {
+      if (!req && !req.params && !req.params.id) {
```

Comment:
Install 'lodash' lib (version 4) and from now on use util functions from lodash.
`const entityId  = _.get(req, 'params.id')`

Reply:
`if (!entityId) return errorHandler({statusCode: 400, message:  `${entityName} ID required.` })`